### PR TITLE
appointments and axios for clients

### PR DESCRIPTION
**Pull request by viktorChubyrka**

### CODE REVIEW

**Comment by viktorChubyrka** [comment]
All is ready

File: src/views/Appointments.vue [hunk 1]
```diff
@@ -1,5 +1,129 @@
...
+
+  methods: {
+    add() {
+      this.appoint.client = this.select._id;
+      axios.post(appointmentsUrl, this.appoint).then(() => {
+        this.fetchClients();
+        this.fetchAppoints();
+      });
+    },
+    loadClientsToSelect() {
```
Comment (user4i) [inline src/views/Appointments.vue]: Really `loadClientsToSelect` method and `clientsSelect` prop are not needed, this code could be realized in select in template.

**Review by user4i — CHANGES_REQUESTED** [review]

File: src/views/Appointments.vue [hunk 2]
```diff
@@ -1,5 +1,116 @@
...
+        res.data.forEach(element => {
+          this.clients.push(element);
+        });
+      });
+    },
+    fetchAppoints() {
+      axios
+        .get(appointmentsUrl + "/" + this.appoint.date.toString())
+        .then(res => {
+          this.appoints = [];
```
Comment (user4i) [inline src/views/Appointments.vue]: `this.appoints = res.data; ` ;)

**Review by user4i — CHANGES_REQUESTED** [review]

File: src/views/Appointments.vue (hunk 1 above)
Comment (viktorChubyrka) [reply src/views/Appointments.vue]: done

File: src/views/Appointments.vue (hunk 2 above)
Comment (viktorChubyrka) [reply src/views/Appointments.vue]: done

**Pull request merged**